blog-pagination-plan.md

```
@@ -0,0 +1,249 @@
+5. Accessible navigation
+
+### Non-Functional Requirements
+1. Maintain site performance (static generation)
+2. Keep styling consistent with existing design
+3. Mobile-responsive pagination
+4. Preserve existing blog post URLs (`/blog/[slug]`)
+
+## Technical Design
+
+### URL Structure
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)